app/code/Vladimirl/Chatter/Model/ResourceModel/Collection/ChatMessageCollection.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Vladimirl\Chatter\Model\ResourceModel\Collection;
+
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+use Vladimirl\Chatter\Model\ChatMessage as Model;
+use Vladimirl\Chatter\Model\ResourceModel\ChatMessage as ResourceModel;
+
+class ChatMessageCollection extends AbstractCollection
```

Comment:
@Vladimir-L Use full class names in `extends`, `implements`, in constructor and in the property definition.